mailpoet/lib/Automation/Integrations/WooCommerce/Payloads/CustomerPayload.php

```diff
@@ -102,6 +102,26 @@ public function getShippingCountry(): ?string {
     return $this->customer->get_shipping_country();
   }
 
+  public function getTotalSpent(): float {
+    if ($this->isGuest()) {
+      return $this->order && $this->order->is_paid() ? (float)$this->order->get_total() : 0.0;
+    }
+    return (float)$this->customer->get_total_spent();
+  }
```

Comment:
Are the float typecasts necessary? Phpstorm is flagging them as redundant, but since the WC methods don't actually have explicit return types I can see why we might want to have these for safety. But in that case we might want to add casts for other things too, like `int` for `get_order_count` below.

Reply:
@johnolek Ahh, I guess I got them copy-pasted or from a Copilot. I'll have a look to unify it one way or another.